tutorials/tutorial11_traffic_lights.ipynb

```diff
@@ -132,16 +142,14 @@
     "phases = [{\"duration\": \"31\", \"state\": \"GrGr\"},\n",
     "          {\"duration\": \"6\", \"state\": \"yryr\"},\n",
     "          {\"duration\": \"31\", \"state\": \"rGrG\"},\n",
-    "          {\"duration\": \"6\", \"state\": \"ryry\"}]\n",
-    "for node_id in nodes:\n",
-    "    tl_logic.add(node_id, tls_type=\"static\", programID=\"1\", offset=None, phases=phases)"
+    "          {\"duration\": \"6\", \"state\": \"ryry\"}]"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Following this step, the TrafficLightParams class should be passed into your scenario as element `traffic_lights`."
+    "In this particular example, each of the 6 intersections corresponds to the same set of possible phases; in other words, at any time, all intersections will be at the same phase. You can, however, customize each traffic light node to have different phases."
```

Comment:
This is a tiny bit hard to parse. I think it is important to make clearer that this is only true for this specific example, and not for the grid in general.